daq/daq.py

```diff
@@ -19,7 +20,7 @@
 LOGGER = logger.get_logger('daq')
 ALT_LOG = logger.get_logger('mininet')
 
-_PID_FILE = 'inst/daq.pid'
+_PID_FILE = os.path.join(DAQ_RUN_DIR, 'daq.pid')
```

Comment:
Some places it's DAQ_RUN and others DAQ_RUN_DIR?  (e.g. bin/dump_network) -- any reason not to make them the same?

Reply:
So in DAQ I'm importing them as xxx_DIR and the env vars are the ones without DIR. I think DAQ_RUN, DAQ_LIB and DAQ_CONF by themselves are not very clear.

Reply:
Got it. So, "internally" (python code) it's XXX_DIR, and then the user-facing surface is just XXX.  That's fine -- just wanted to make sure the user-facing surface was consistent.